src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.head {
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.inner__container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  position: relative;
+  bottom: 1px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+  margin-right: 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+  margin-right: 50px;
+  list-style: none;
+}
+
+.refer:link {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  padding: 22.5px 0;
+}
+
+.refer:visited {
+  color: #00acdc;
+}
+
+.refer:hover {
+  color: #00acdc;
+}
+
+.nav__link :first-child .refer {
+  margin: 0 10px 0 0;
+}
+
+.nav__link :nth-child(1n+8) {
+  margin: 0 0 0 10px;
+}
+
+.nav__link :nth-child(1n+2):not(:last-child) {
+  margin: 0 10px;
+}
+
+.is-active:link {
+  color: #00acdc;
+}
+
+.refer:active::after,
+.nav__link :first-child .refer::after {
```

Comment:
simplify it, you dont need such specific selectors

Reply:
sorry, could you explain it to me, please ? I don't understand exactly what you mean

Reply:
For future. If you need to set some properties to only one element, use a class or modifier. `:first-child` is usually used when you need to set something to each first element in some class or reused blocks.